std/traits.d

```diff
@@ -6294,7 +6294,7 @@ template isSomeFunction(T...)
 
 /**
 Detect whether $(D T) is a callable object, which can be called with the
-function call operator $(D $(LPAREN)...$(RPAREN)).
+function call operator `(...)`.
```

Comment:
I don't think the LPAREN RPAREN is even necessary in ddoc, it can handle matched parens.